.claude/hooks/protect-claude-dir.sh

```
@@ -0,0 +1,47 @@
+#!/bin/bash
+# PreToolUse hook to explain why .claude directory access is forbidden
+#
+# This hook provides a helpful error message when Claude attempts to
+# read, edit, or write files in the .claude directory.
+
+input=$(cat)
+
+cd "$CLAUDE_PROJECT_DIR"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)